Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.16-alpine
```

Comment:
I don't see version 1.16 in the [list of supported images](https://hub.docker.com/_/golang). It looks like we should be using 1.18.

Furthermore, the `-alpine` variant is experimental and unsupported apparently:
> This variant is highly experimental, and not officially supported by the Go project (see [golang/go#19938](https://github.com/golang/go/issues/19938) for details).

Reply:
fixed